llvm/test/TableGen/GlobalISelEmitterVariadic.td

```diff
@@ -1,4 +1,5 @@
 // RUN: llvm-tblgen -gen-global-isel -I %p/../../include -I %p/Common %s -o - | FileCheck %s
+// RUN: llvm-tblgen -gen-global-isel -I %p/../../include -I %p/Common %s -o - -DIGNORE | FileCheck %s --check-prefix=CHECK0
```

Comment:
Should add -warn-on-skipped-patterns and make sure no warning is produced in stderr

Reply:
These records naturally do not produce a warning with `-warn-on-skipped-patterns` with or without `GISelShouldIgnore` though. This test just checks to see if the matcher omits them.

In the other test [test/TableGen/GlobalISelEmitterSkippedPatterns.td](https://github.com/llvm/llvm-project/pull/88382/files#diff-04bf1987d3970e5d2033f31034e8cb1f4ff40ab85db1f25aad8490ec25af9895), no record is generated in the MatchTable naturally, so that one simply checks for the warning.

Reply:
The entire point of the feature is to silence the warning for cases we don't want to import though

Reply:
There is a check for warning omission in `GlobalISelEmitterSkippedPatterns.td`.

The .inc check test can be removed. (Alternatively, a test that skips the import/warning in global yet keeps it within the Table in dag, so that we can check both .inc and warning in the same file could be useful.) Should I just remove it or add a new test (with or without .inc check)? We have the warning check already.